src/Directory.Build.props

```diff
@@ -5,6 +5,11 @@
   <!-- This file is imported by all projects at the beginning of the project files -->
   <Import Project="$(RepoToolsetDir)Settings.props" />
 
+  <PropertyGroup>
+    <IncludeSymbols>true</IncludeSymbols>
```

Comment:
Cool, I'll try that. Sounds like it doesn't matter in this case, but embedded PDBs would increase file size, right?